cmflib/cli/parser.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import logging
+import os
+import sys
+
+from cmflib.cli import CmfParserError
+from cmflib.commands import (
```

Comment:
Please check, I think the folder is name 'command' not 'commands'

Reply:
Made the change